utils/flare.py

```diff
@@ -188,6 +209,10 @@ def upload(self, email=None):
             'files': {'flare_file': open(self.tar_path, 'rb')},
             'timeout': self.TIMEOUT
         }
+
+        self.set_proxy(requests_options)
+        self.set_ssl_validation(requests_options)
+
         if Platform.is_windows():
             requests_options['verify'] = os.path.realpath(os.path.join(
```

Comment:
Could you move this `if` block to your `set_ssl_validation` method, before you set `verify` depending on the value of `skip_ssl_validation`?

We need to make sure that we'll also skip ssl validation on Windows when the configuration asks for it.

Thanks!